src/components/JobLogs/JobLogs.js

```diff
@@ -63,18 +81,21 @@ export class JobLogs extends Component {
       return <Preloader />;
     }
 
+    const contentDataWithLog = contentData.map(item => assign(item, { log: this.createLogButton(item.jobExecutionId) }));
+
     return (
       <MultiColumnList
         id="job-logs-list"
-        totalCount={contentData.length}
-        contentData={contentData}
+        totalCount={contentDataWithLog.length}
+        contentData={contentDataWithLog}
         columnMapping={this.columnMapping}
         visibleColumns={this.visibleColumns}
         formatter={formatter}
         sortOrder={sortField}
         sortDirection={sortDirection}
         autosize
         onHeaderClick={onSort}
+        columnWidths={{ log: 80 }}
```

Comment:
Move to the constructor and name as `columnWidths` like there is already for `columnMapping`.